tests/EmailTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace emailApp;
+include dirname(__FILE__)."/../vendor/autoload.php";
+
+class EmailTest extends \PHPUnit_Framework_TestCase {
+	
+	/**
+	 *@test
+	 */
+	public function it_can_validate_email() {
+		$object = new Email();
+		$result = $object->validateEmail("shantanu3687@gmail.com");
+		$this->assertEquals($result, 1);
+	}
+    /**
+	 *@test
+	 *
+	 */
+	public function it_can_validate_email_starting_with_number() {
+		$object = new Email();
+		$result = $object->validateEmail("111shantanu3687@gmail.com");
+		$this->assertEquals($result, 1);
+	}
+	/**
+	 *@test
+	 * @expectedException InvalidArgumentException
+	 */
+	public function it_should_throgh_InvalidArgumentException_for_email_having_more_number_of_at_the_rate_sign() {
```

Comment:
`@` is called `at sign` https://en.wikipedia.org/wiki/At_sign

In test case title you are saying more than one at signs. but in actual test case you are passing just one. and more dots.